### DISCUSSION TITLE
'autoUpdate' is not exported from '@floating-ui/react-dom'

### DISCUSSION THREAD

**Discussion by matobiely** [floating-ui/floating-ui#1567] [Q&A]
Why i am getting this error, when https://floating-ui.com/docs/react-dom#updating here is defined, that it should be exported from react-dom. 

thank you

**Comment by atomiks** [comment]
<img width="673" alt="Screen Shot 2022-03-04 at 1 00 02 am" src="https://user-images.githubusercontent.com/22450188/156579442-a019adf3-98a9-40e6-b15a-00578a8461a5.png">

Make sure you're on `0.5.0`, it's a new feature
